CONTRIBUTING.md

```diff
@@ -1,30 +1,30 @@
-If you want to contribute to Theano, have a look at the instructions here:
-http://deeplearning.net/software/theano/dev_start_guide.html
+If you want to contribute to Aesara, have a look at the instructions here:
+http://deeplearning.net/software/aesara/dev_start_guide.html
 
 
-## Migrating PRs from original Theano Repo
-Theano-PyMC is actively merging new changes. If you have a pull request on the original respository and would like to move it here use the following commands in your local theano repo
+## Migrating PRs from original Aesara Repo
+Aesara-PyMC is actively merging new changes. If you have a pull request on the original respository and would like to move it here use the following commands in your local aesara repo
 
 ```
-# Go to your Theano Repo
+# Go to your Aesara Repo
 cd /path/to/your/repo
 
-# If you'd like to add theano-PyMC as a remote
-git remote add pymc git@github.com:pymc-devs/Theano-PyMC.git
+# If you'd like to add aesara-PyMC as a remote
+git remote add pymc git@github.com:pymc-devs/Aesara-PyMC.git
```

Comment:
Yeah, regarding the broken links, I changed them intentionally I thought it would be kinda motivating to reset everything and force ourselves for using Aesara name. 
I could be wrong :/ I think the best strategy now is to discuss this PR further...